### ISSUE TITLE
Use the compendium browser to add a feat to the ASI advancement

### ISSUE THREAD

**Issue by kaelad02** [foundryvtt/dnd5e#4665]
When you use the ASI advancement, there's an option to allow a feat instead. There's a space at the bottom of the advancement window where you can drag the feat onto. Can we make that clickable so it brings up the Compendium Browser so the player can select a feat from there? Kind of like how clicking on the blank space for the Species on the character sheet brings up the Compendium Browser to select one.

**Comment by arbron** [comment]
Yeah, integrating the compendium browser into advancements where useful will hopefully come in `4.2` when I redesign the advancement UI using the new styles.
